test/bigtest/helpers/mountWithContext.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Harness from './Harness';
+import getCleanTestingRoot from './getCleanTestingRoot';
+
+export function mountWithContext(component) {
```

Comment:
See above comment about copying-in code that can be imported from stripes-components.